src/component/category.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react";
+
+  const Category = () => {
```

Comment:
Change the name of the component to Capital at first
and make your code cleaner and more readable --the indentation as specific